Tools/Scripts/generate-gpup-webgl

```diff
@@ -124,6 +124,9 @@ messages -> RemoteGraphicsContextGL NotRefCounted Stream {{
     void MultiDrawElementsInstancedANGLE(uint32_t mode, IPC::ArrayReferenceTuple<int32_t, int32_t, int32_t> countsOffsetsAndInstanceCounts, uint32_t type)
     void MultiDrawArraysInstancedBaseInstanceANGLE(uint32_t mode, IPC::ArrayReferenceTuple<int32_t, int32_t, int32_t, uint32_t> firstsCountsInstanceCountsAndBaseInstances)
     void MultiDrawElementsInstancedBaseVertexBaseInstanceANGLE(uint32_t mode, IPC::ArrayReferenceTuple<int32_t, int32_t, int32_t, int32_t, uint32_t> countsOffsetsInstanceCountsBaseVerticesAndBaseInstances, uint32_t type)
+#if PLATFORM(COCOA)
+    void CreateEGLSync(MachSendRight syncEvent, uint64_t signalValue) -> (uint64_t sync) Synchronous NotStreamEncodable NotStreamEncodableReply
```

Comment:
See above, this shouldn't be needed when the function CreateEGLSync is declared in the autogen section.

Reply:
I had a lot of trouble using the autogen for this function because of `MachSendRight`. I need to add `NotStreamEncodable NotStreamEncodableReply` which I don't think I can do with the autogen methods.